holidays/constants.py

```diff
@@ -54,6 +54,7 @@
 HEBREW = "hebrew"
 HINDU = "hindu"
 ISLAMIC = "islamic"
+ORTHODOX = "orthodox"
```

Comment:
It looks like we don't use `CHRISTIAN` and for v1 `CATHOLIC` should be `CHRISTIAN_CATHOLIC`, `ORTHODOX` - `CHRISTIAN_ORTHODOX`

Or it's too much?

Reply:
It sounds rational. 👍